README.md

```diff
@@ -11,6 +11,8 @@ A teaching platform for computer-aided drug design (CADD) using open source pack
 [![GH Actions CI ](https://github.com/volkamerlab/teachopencadd/workflows/CI/badge.svg)](https://github.com/volkamerlab/teachopencadd/actions?query=branch%3Amaster+workflow%3ACI)
 [![GH Actions Docs](https://github.com/volkamerlab/teachopencadd/workflows/Docs/badge.svg)](https://projects.volkamerlab.org/teachopencadd/)
 
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/teachopencadd/badges/installer/conda.svg)](https://conda.anaconda.org/conda-forge)
```

Comment:
Thanks for adding our TOC conda badge!! Could you double-check the link, please? I am directed to https://anaconda.org/conda-forge/repo?type=conda&label=main


Maybe it needs to be this?
```suggestion
[![Anaconda-Server Badge](https://anaconda.org/conda-forge/teachopencadd/badges/installer/conda.svg)](https://anaconda.org/conda-forge/teachopencadd)
```

Reply:
Of course, sorry about the typo!